File: 25-06-2021/UnActivityAOtra/app/src/main/java/com/example/unactivityaotra/MainActivity.java
package com.example.unactivityaotra;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
    public void siguiente( View V ) {
        Intent siguiente = new Intent(this, MainActivity3.class);
        startActivity(siguiente);
    }
}

File: KotlinToDo/app/src/main/java/com/example/kotlintodo/MainActivity.kt
package com.example.kotlintodo

import android.content.Context
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.view.inputmethod.InputMethodManager

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }
    private fun ocultarTeclado()
    {
        val view : View? = this.currentFocus
        if(view != null){
            val manager : InputMethodManager = getSystemService(
                Context.INPUT_METHOD_SERVICE
            ) as InputMethodManager
            manager.hideSoftInputFromWindow(
                view.windowToken,0
            )
        }
    }
}